cromshell

```diff
@@ -321,6 +329,26 @@ function invalidSubCommand()
   exit 3;
 }
 
+function getAnswerFromUser()
+{
+  local prompt="${1}" 
+  local acceptableValues="${2}"
+  local responseVar="${3}" 
+
+  local haveGoodValue=false
+  while ! $haveGoodValue ; do
+    read -p "${prompt} [$( echo ${acceptableValues} | sed 's# #/#g' )] : " ${responseVar}
+
+    for okVal in ${acceptableValues} ; do
```

Comment:
can we make this case insensitive?  ` | tr a-z A-Z` according to stack overflow

Reply:
Sure.